templates/vue/src/components/Lightbox/index.vue

```diff
@@ -297,6 +301,9 @@ export default {
         height: this.lightboxDimensions.height,
       }
     },
+    handleUnitChange(newNode) {
```

Comment:
Since this is the node ID, it's better to name it `nodeId` or `selectedNodeId` rather than `newNode`